plugin/completion.py

```diff
@@ -41,10 +42,72 @@
 
 
 class LspResolveDocsCommand(sublime_plugin.TextCommand):
+    def run(self, edit: sublime.Edit, index: int) -> None:
+        item = CompletionHandler.completions[index]
+        detail = minihtml(item.get('detail') or "", self.view)
+        documentation = minihtml(item.get("documentation") or "", self.view)
+
+        # don't show the detail in the cooperate AC popup if it is already shown in the AC details filed.
+        self.is_detail_shown = bool(detail)
+
+        content = self.get_content(documentation, detail)
+        self.show_popup(content)
+
+        if not detail or not documentation:
+            # To make sure that the detail or documentation fields doesn't exist we need to resove the completion item.
+            # If those fields appear after the item is resolved we show them in the popup.
+            self.do_resolve(item)
+
+    def get_content(self, documentation: str, detail: str) -> str:
+        content = ""
+        if detail and not self.is_detail_shown:
+            content += "<div class='highlight' style='margin: 4px'>{}</div>".format(detail)
+
+        if documentation:
+            content += "<div style='margin: 10px'>{}</div>".format(documentation)
+
+        return content
+
+    def show_popup(self, content: str) -> None:
+        mdpopups.show_popup(
+            self.view,
+            content,
+            flags=sublime.COOPERATE_WITH_AUTO_COMPLETE,
+            max_width=480,
+            max_height=300,
+            allow_code_wrap=True
+        )
+
+    def do_resolve(self, item: dict) -> None:
+        session = session_for_view(self.view, 'completionProvider', self.view.sel()[0].begin())
```

Comment:
Since #1030 was integrated once you rebase you can directly check capability `'completionProvider.resolveProvider'` and skip capability checks below.

Reply:
16741d06ce4b4b6c1da0073e743543882719bf94 is this how you imagined it?

Reply:
I'm not gonna say this is wrong because I don't want to open another big exchange here but sending requests to all servers seems wrong. Sending it just to the first one also seems wrong. Ideally, we would send it to the one that provided the suggestion.

Maybe let's stick to sending the request just to the first server that supports capability?
After you rebase, the capability argument will also exist for `session_for_view` (singular).

Reply:
This is probably something we need to discuss in a separate issue and in more detail.

Reply:
@rchl I had to revert back to the old way of checking the capability.
I left a brief description in the commit message describing the problem.

Can you take a look?
6d7103e999c09defc4862416ad005bde469d72ea